package.json

```
@@ -36,7 +36,8 @@
     "generate:locales": "ts-node --skip-project -O '{\"lib\":[\"es2015\"]}' scripts/make-locales.ts",
     "generate:content-definitions": "rimraf definitions/content && mkdir -p definitions/content && gen-api-models --api-spec $npm_package_io_content_specs --out-dir ./definitions/content",
     "generate:all": "npm-run-all generate:mock-google-services-json generate:api-definitions generate:pagopa-api-definitions generate:content-definitions generate:bonus-vacanze-definitions generate:bpd-citizen generate:locales",
-    "locales_unused": "ts-node --skip-project -O '{\"lib\":[\"es2015\"]}' scripts/unused-locales.ts"
+    "locales_unused": "ts-node --skip-project -O '{\"lib\":[\"es2015\"]}' scripts/unused-locales.ts",
+    "jsDocs": "jsdoc2md ts/utils/webviewScripts/*.js > ts/utils/webviewScripts/README.md"
```

```suggestion
    "generate-myportal-readme": "jsdoc2md ts/utils/webviewScripts/*.js > MYPORTAL_README.md"
```

- kebab-case is [preferable](https://it.wikipedia.org/wiki/Kebab_case) 
- a better name could help to know what that command does
- avoid to include no-code files into the codebase. The root should be fine